examples/blueprint-construct/index.ts

```diff
@@ -65,14 +72,14 @@ export default class BlueprintConstruct extends cdk.Construct {
             new ssp.addons.VpcCniAddOn(),
             new ssp.addons.CoreDnsAddOn(),
             new ssp.addons.KubeProxyAddOn(),
-            new ssp.addons.OpaGatekeeperAddOn()
+            new ssp.addons.OpaGatekeeperAddOn(),
+            new ssp.addons.KarpenterAddOn({defaultProvisionerSpecs: provisionerSpecs}),
```

Comment:
We typically don't provide any addOn parameters in this example. Will the addOn work without any parameters?

Reply:
agree. The provisioner can be used in a pattern.

Reply:
Will add provisioner to the patterns. Removed.